app/components/SuggestionContents/index.jsx

```diff
@@ -0,0 +1,81 @@
+import TextDiff from '../TextDiff'
+import React from 'react'
+import { IntlMixin } from 'react-intl'
+import cx from 'classnames'
+
+/**
+ * Display all content strings (singular or plurals) for a suggestion.
+ * May show a diff against a set of provided strings.
+ */
+let SuggestionContents = React.createClass({
+  mixins: [IntlMixin],
+
+  propTypes: {
+    plural: React.PropTypes.bool.isRequired,
+    contents: React.PropTypes.arrayOf(
```

Comment:
It is required on purpose so that someone can't use the component, forget to pass in that prop, then be confused about why it doesn't look right. I could leave it out, but then I get no help from React if I forget to include it.